packages/react-widgets/__tests__/layers/FeatureSelectionLayer.test.js

```diff
@@ -0,0 +1,21 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { EventManager } from 'mjolnir.js';
+import FeatureSelectionLayer from '../../src/layers/FeatureSelectionLayer';
+import { mockReduxHooks } from '../mockReduxHooks';
+
+describe('FeatureSelectionLayer', () => {
+  // mockReduxHooks();
+  describe('Google Maps compatibility', () => {
```

Comment:
Not sure if just in raster?

```suggestion
  describe('Google Maps Raster compatibility', () => {
```